releaseNotes/AzureDevOpsPushPipelineExtension.md

```diff
@@ -1,6 +1,7 @@
 ## Azure App Configuration Push Extension release notes
-Azure App Configuration Push extension for Azure DevOps pipeline can be installed from the [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task-push). The version of the Azure App Configuration task will print to the console when the task is executed: ![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)
+Azure App Configuration Push extension for Azure DevOps pipeline can be installed from the [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task-push). The version information will print to the console when the task is executed:

```

Comment:
>will print [](start = 252, length = 10)

`will print` => `is printed` OR `will be printed`

Reply:
Updated.